keel-api/src/test/resources/examples/titus-cluster-example.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: titus.spinnaker.netflix.com/v1
+kind: titus-cluster
+metadata:
+  serviceAccount: my-email@spinnaker.io
+spec:
+  moniker:
+    app: keeldemo
+    stack: examples
+    detail: ec2v1
+  container:
+    registry: testregistry
+    organization: emburns
+    image: emburns/spin-titus-demo
+    digest: ""
+    tag: latest
+  locations:
+    account: titustestvpc
+    regions:
+      - name: us-west-2
+      - name: us-east-1
+  containerOptions:
```

Comment:
Any particular reason to have this separate from the `container` key?

Reply:
ehh i'm not sure. I was thinking that `container` is meant to be specifying the image, whereas the other options were..different? Maybe that's incorrect or weird. Asher suggested pulling everything under `containerOptions` up to the same level as everything else, instead of nesting it. I think I like that.